apps/frontend/src/lib/storage/localStorage/user/connection/message/index.ts

```diff
@@ -138,14 +97,136 @@ export const processNewMessages = async (
     }
   );
 
+  return newBackupEntries;
+}
+
+async function handleEdges(session: Session, user: User, messages: EdgeMessage[]): Promise<CreateBackupData[]> {
+  let tappedHash: string = "";
+  if (user && user.tapGraphEnabled) {
+    // Double hash the signature private key, use as identifier, use single hash version as revocation code
+    tappedHash = sha256(sha256(user.signaturePrivateKey));
```

Comment:
same here, tappedHash = sha256(sha256(user.signaturePrivateKey) || "DEVCON2024");

Reply:
Ah I see, is this so a single key can map to multiple issuer graphs?

Reply:
yeah, i think the edgeId being universal is fine, but anything we do for the devcon graph should be specific to devcon